### PR TITLE
Added notification for Walmart groceries - pickup and delivery.

### PR DESCRIPTION
**Pull request by parikhv15**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,20 +109,20 @@ https://github.com/jvkumar/delivery-finder/blob/master/assets/wholesfood2_1.png)
 *IMPORTANT* - Make sure you don't close the retailer's checkout page (from where you collected the cookie) ever while this finder is running. So let the page remain opened.
 
 ## Step 5
-Go to `covid19` folder and then subfolder `delivery-finder` subfolder where this finder codebase is residing. 
+Go to `covid19` folder and then subfolder `delivery-finder` subfolder where this finder codebase is residing.
 
 ## Step 6
```
Comment (jvkumar) [inline README.md:125]: @parikhv15 since you are moving cookie configuration in `config/auth.js ` so step 6 would need change as well.

**Comment by jvkumar** [comment]
@parikhv15 Thank you for arranging the code to make it much modular. I appreciate your time improving it that I left to do in the future (my procrastination).

**Comment by parikhv15** [comment]
@jvkumar You are most welcome! I really appreciate your time initiative in making this simple. Just a day before this I was thinking of having something like this, but did not do anything. So, all thanks to you.

I will edit the instructions and make a push.

**Comment by jvkumar** [comment]
👍

**Pull request merged**